bin/SuperFluxNN

```diff
@@ -103,7 +104,9 @@ def main():
         pp = PeakPickingProcessor(**vars(args))
         from madmom.utils import write_events as writer
         # sequentially process them
-        out_processor = [rnn, pp, writer]
+        # Note: we need np.atleast_2d and np.transpose before the RNN, since
+        #       it expects the data in 2D (1D means framewise processing)
+        out_processor = [np.atleast_2d, np.transpose, rnn, pp, writer]
```

Comment:
Yes and no. I thought about this as well, but opted for the way of least invasion to existing code.
As you state, returning a 2d-array is a bit unintuitive.

There's a TODO item in #185 which says: "rewrite all features as processors (mostly `madmom.features.onsets.*`)". I think it is better to address this separately and not in this PR.